### DISCUSSION TITLE
How to handle file changes on remote

### DISCUSSION THREAD

**Discussion by fsfettel** [Natizyskunk/vscode-sftp#226] [Q&A] [answered]
I have the issue, that sometimes files are changed on the server-side during my changes. 

How can i configure, that to prevent vscode-ftp from overwriting remote side or get a warning or something?

**Comment by Natizyskunk** [comment, marked as answer]
@fsfettel Sadly there's no warning if the last modified date of the remote file is more recent as your local file.

For now I can only recommend you to use the command `SFTP: Diff Active File with Remote` from the command palette :

![image](https://github.com/Natizyskunk/vscode-sftp/assets/59120131/fb8eab01-3cc3-426e-ad07-b991ea87331e)

Also if you prefer, you can make a backup of your local file, download the latest version available on the remote and then merge your backup with the new downloaded file before uploading it again to the remote.
